man/en/sos-report.1

```diff
@@ -317,6 +317,16 @@ output, or string data from the system. The resulting logs may be used
 to understand the actions that sos would have taken without the dry run
 option.
 .TP
+.B \--estimate-mode
+Estimate disk space requirements when running sos report. This can be valuable
+to prevent sosreport working dir to consume all free disk space. No plugin data
+is available at the end.
+
+Plugins will be collected sequentially, size of collected files and commands outputs
+will be calculated and the plugin files will be immediatelly deleted prior execution
+of the next plugin. This still can consume whole free disk space, though. Also the
+estimation might not be accurate in several ways.
```

Comment:
Last bit could use a little more detail. Perhaps "Please note, size estimations may not be accurate for highly utilized systems due to changes between an estimate and real execution".